test/integration/scheduler/filters/filters_test.go

```diff
@@ -2485,6 +2487,7 @@ func TestUnschedulablePodBecomesSchedulable(t *testing.T) {
 // even if turing the feature gate enabled or disabled.
 func TestPodAffinityMatchLabelKeyEnablement(t *testing.T) {
 	// enable the feature gate
+	featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.32"))
```

Comment:
emulating 1.32 shouldn't be required if we want the feature enabled

Reply:
The feature gate is going to get disabled later within this test.